client/src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Login.scss';
+
+import { TextInput } from '../../components/input/TextInput/TextInput';
+import { Button } from '../../components/button/Button/Button';
+
+import BrachioR from '../../assets/login/brachiosaurus-ground-right.svg';
+import BrachioL from '../../assets/login/brachiosaurus-ground-left.svg';
+import Ground from '../../assets/login/ground.svg';
+import MountainB from '../../assets/login/mountain-back.svg';
+import MountainFL from '../../assets/login/mountain-front-left.svg';
+import MountainFR from '../../assets/login/mountain-front-right.svg';
+import MountainM from '../../assets/login/mountain-mid.svg';
+
+const PageLogin = ({ incorrectEntry }) => {
+  return (
+    <div className="login-entire-page">
+      <div className="login-bg">
+        <div className="login-container">
+          <h2 className="login-title">Login</h2>
+          <TextInput inputType={'text'} placeholder={'Username or Email'}></TextInput>
+          <TextInput
+            inputType={'password'}
+            placeholder={'Password'}
+            errorFeedback={'Invalid username or password. Please try again!'}
+          ></TextInput>
+
+          {/* Invalid Username or Password */}
+          {/* {incorrectEntry ? <p className="login-error-message">Incorrect Username or Password, please try again!</p> : <></>} */}
+
+          <div className="login-button-container">
+            <p className="forgot-message">Forgot Password?</p>
+            <Button label={'Log in'}></Button>
```

Comment:
When the login button is pressed it should check the credentials and produce an error message
If you need help with this let me know, it requires state management
And to set the error if one is produced from the backend

Also put your functions in a separate file
(see https://github.com/UofT-Frosh-Orientation/orientation-website/pull/125 for e.g. of separate file functions)